README.md

```diff
@@ -1,24 +1,195 @@
 
-* Ruby version
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deploy)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* System dependencies
+<!-- PROJECT DESCRIPTION -->
 
-* Configuration
+# 📖 Budget-Wise🍜<a name="about-project"></a>
 
-* Database creation
+> The Budget-Wise app is a great application to plan your expenses. You can sign up and start generating your transactions and budgets
 
-* Database initialization
+![](splash.png)
 
-* How to run the test suite
+## 🛠 Built With <a name="built-with"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+### Tech Stack <a name="tech-stack"></a>
 
-* Deployment instructions
+> Here are List of technologies used on this project
 
-* ...
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://ruby-lang.org/en">Ruby</a></li>
+    <li><a href="https://javascript.com/">JavaScript</a></li>
+    <li><a href="https://html.spec.whatwg.org/multipage/">HTML</a></li>
+    <li><a href="https://www.w3.org/TR/CSS/#css">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Linters</summary>
+  <ul>
+    <li><a href="https://www.rubocop.org/">Rubocop</a></li>
+    <li><a href="https://www.stylelint.io/">Stylelint</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Test library</summary>
+  <ul>
+    <li><a href="https://www.rspec.info/">Rspec</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Create budget
+- Generate transactions
+- Keeps track of all your expenses
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Ruby installed on your computer
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+  git clone https://github.com/TracyMuso/Budget-Wise.git
+```
+
+change directory to cloned folder
+
+```
+  cd Budget-Wise
+```
+
+### Install
+
+Install project dependecies with:
+
+```
+  bundle install
+```
+
+### Tests <a name="run-tests"></a>
+```
+   rspec
+```
+
+ ### Deployment <a name="deploy"></a>
+
+ - [Live demo](https://budegt.onrender.com)
+ - [Presentation video](https://www.loom.com/share/e39605e9ae5342e8ac64a5bdbd59fb10)
```

Comment:
[REQUIRED] Your video doesn't contain all of your app's major features and it doesn't highlight a piece of code that you find interesting.